### PR TITLE
add goodbye message when user logs out

### LINKED ISSUE #8: Show "goodbye" message when user logs out

### PR DESCRIPTION
**Pull request by cjdufault**
@yp0975eu @NateBigStone @ashley7295 @sunnycake 
Added a "goodbye" message that is displayed when a user logs out.
Closes #8

### CODE REVIEW

File: lmn/views/views_users.py [hunk 1]
```diff
@@ -40,3 +40,9 @@ def register(request):
 
     form = UserRegistrationForm()
     return render(request, 'registration/register.html', {'form': form} )
+
+
+def logout_user(request):
+    username = request.user.username
+    logout(request)
+    return render(request, 'registration/logout.html', { 'username': username })
```
Comment (NateBigStone) [inline lmn/views/views_users.py:48]: This file doesn't seem to exist

**Review by NateBigStone — CHANGES_REQUESTED** [review]

File: lmn/views/views_users.py (hunk 1 above)
Comment (NateBigStone) [reply lmn/views/views_users.py:48]: django.template.exceptions.TemplateDoesNotExist: registration/logout.html

**Comment by yp0975eu** [comment]
This needs some sort of test and it should be good to go.

File: lmn/views/views_users.py (hunk 1 above)
Comment (cjdufault) [reply lmn/views/views_users.py:48]: Sorry, I had that locally and forgot to add it.

**Review by yp0975eu — CHANGES_REQUESTED** [review]
For some reason `views_notes.py` and `base_styles.css` are being overwritten. Can you clean up the pull request to only have actual changes?

**Comment by cjdufault** [comment]
@yp0975eu I'm not sure what happened there, but the versions of both those files in both branches are identical. It won't actually make any changes when merged.

**Comment by yp0975eu** [comment]
It will make changes. It will overwrite the file's history showing this commit as it's origin and that is not entirely true.

**Review by NateBigStone — APPROVED** [review]

**Comment by cjdufault** [comment]
@yp0975eu I'm not totally sure how to go about doing that. Could you explain it? Frankly, it seems like delving into a mess when there isn't a real problem (this won't change a single line of code), but if you can explain what you want done, I'll do it.

**Comment by yp0975eu** [comment]
Please `$git rm /views/views_notes.py`

**Comment by yp0975eu** [comment]
Also, please `$git rm static/css/base_style.css`.

**Review by yp0975eu — CHANGES_REQUESTED** [review]
This appears to be changing the directory structure.  

lmn/static/css/base_style.css → static/css/base_style.css
lmn/views/views_notes.py → views/views_notes.py

File: views/views_notes.py [hunk 2]
```diff
@@ -23,14 +23,14 @@ def new_note(request, show_pk):
...
-    else :
+    else:
         form = NewNoteForm()
 
     return render(request, 'lmn/notes/new_note.html' , { 'form': form , 'show': show })
 
 
 def latest_notes(request):
-    notes = Note.objects.all().order_by('-posted_date')
+    notes = Note.objects.all().order_by('-posted_date')[:20]   # the 20 most recent notes
```
Comment (yp0975eu) [inline views/views_notes.py:33]: Where did this come from?

File: static/css/base_style.css [hunk 3]
```diff
@@ -15,6 +15,9 @@ a {
 }
 
 .error {
-  color: #e60c42;
-  font-weight: bold;
```
Comment (yp0975eu) [inline static/css/base_style.css:19]: Are these supposed to gone?

File: views/views_notes.py (hunk 2 above)
Comment (cjdufault) [reply views/views_notes.py:33]: No idea, I didn't do that.

File: static/css/base_style.css (hunk 3 above)
Comment (cjdufault) [reply static/css/base_style.css:19]: I didn't touch this file.

File: views/views_notes.py (hunk 2 above)
Comment (NateBigStone) [reply views/views_notes.py:33]: @yp0975eu maybe @cjdufault 's remote fetch is clara's repo?

https://github.com/claraj/lmn/blame/master/lmn/views/views_notes.py#L33

Comment (yp0975eu) [reply views/views_notes.py:33]: @NateBigStone , That would explain it.

Comment (NateBigStone) [reply views/views_notes.py:33]: @cjdufault what does `git remote -v` show?

Comment (NateBigStone) [reply views/views_notes.py:33]: Regardless, I think this file is how it ought to be.

**Pull request closed without merging**